import scala.scalanative.unsafe.*, stdio.*

object Hello {
  def main(args: Array[String]): Unit = {
    throw new Exception()
  }
}
